app/controllers/hyrax/generic_works_controller.rb

```diff
@@ -99,6 +99,24 @@ def identifiers
     end
   end
 
+  def tombstone
+    #Make it so work does not show up in search result for anyone, not even admins.
+    curation_concern.visibility = Hydra::AccessControls::AccessRight::VISIBILITY_TEXT_VALUE_PRIVATE
+    curation_concern.depositor = 'TOMBSTONE-' +   curation_concern.depositor
+
+    #I tried this but it did not work.  And gets complicated.
+    #curation_concern.state = Vocab::FedoraResourceStatus.inactive
+
+    for i in 0 ... curation_concern.file_sets.size
+       curation_concern.file_sets[i].visibility = Hydra::AccessControls::AccessRight::VISIBILITY_TEXT_VALUE_PRIVATE
+    end 
+
```

Comment:
curation_concern.file_sets.each { |f| f.visibility = Hydra::AccessControls::AccessRight::VISIBILITY_TEXT_VALUE_PRIVATE }

I think I got the syntax right, ask Bill :)